features/cart/shopping_cart/changing_dispalay_currency_of_cart.feature

```diff
@@ -1,8 +1,8 @@
 @shopping_cart
-Feature: All of my cart's values get updated to the currency of my choosing
-    In order to know how much I have to pay in my currency
+Feature: Changing display currency of cart
+    In order to know estimated price for foreign currencies
     As a Visitor
-    I want to see every cash amount in my chosen currency
+    I want to see rounded every cash amount in my chosen currency
```

Comment:
`Changing display currency of the cart`
and
`I want to see every cash amount rounded to my chosen currency`
or something alike